requirements-test.txt

```diff
@@ -1,8 +1,8 @@
+celery>=4.4.0,<5.0
```

Comment:
remove this, it's already defined in openwisp-controller which is a dependency of this package